### PR TITLE
some small bug fix and improvement usability

### PR DESCRIPTION
**Pull request by kassyi**
1. update Authenticator app url.
2. add autocomplete="off" to login from

### CODE REVIEW

File: google-authenticator.php [hunk 1]
```diff
@@ -312,15 +312,15 @@ function user_setup_page() {
 		),
 		array(
 			'text' => __( 'iOS: Google Authenticator', 'google-authenticator' ),
-			'link' => 'https://itunes.apple.com/app/authy/id494168017',
+			'link' => 'https://itunes.apple.com/us/app/google-authenticator/id388497605',
 		),
 		array(
 			'text' => __( 'Android: Authy', 'google-authenticator' ),
-			'link' => 'https://itunes.apple.com/app/authy/id494168017',
+			'link' => 'https://play.google.com/store/apps/details?id=com.authy.authy',
```
Comment (ivankruchkoff) [inline google-authenticator.php]: Good catch

File: google-authenticator.php [hunk 2]
```diff
@@ -312,15 +312,15 @@ function user_setup_page() {
...
 		),
 		array(
 			'text' => __( 'Android: Google Authenticator', 'google-authenticator' ),
-			'link' => 'https://itunes.apple.com/app/authy/id494168017',
+			'link' => 'https://play.google.com/store/apps/details?id=com.google.android.apps.authenticator2',
```
Comment (ivankruchkoff) [inline google-authenticator.php]: Another good catch 🤦‍♂️

File: google-authenticator.php [hunk 3]
```diff
@@ -312,15 +312,15 @@ function user_setup_page() {
 		),
 		array(
 			'text' => __( 'iOS: Google Authenticator', 'google-authenticator' ),
-			'link' => 'https://itunes.apple.com/app/authy/id494168017',
+			'link' => 'https://itunes.apple.com/us/app/google-authenticator/id388497605',
```
Comment (ivankruchkoff) [inline google-authenticator.php]: I think it's better to leave the original link, as not everyone is using the US store.

File: google-authenticator.php (hunk 1 above)
Comment (kassyi) [reply google-authenticator.php]: thx!

File: google-authenticator.php (hunk 2 above)
Comment (kassyi) [reply google-authenticator.php]: thx!

File: google-authenticator.php (hunk 3 above)
Comment (kassyi) [reply google-authenticator.php]: sorry, I was wrong. original link is better.

Comment (ivankruchkoff) [reply google-authenticator.php]: I think based on the reviews, the authy one is better, or we could do the same as for Android and link to both google authenticator and authy and let the end user decide

Comment (kassyi) [reply google-authenticator.php]: at source code, iOS authy link is exits.

`array(
			'text' => __( 'iOS: Authy', 'google-authenticator' ),
			'link' => 'https://itunes.apple.com/app/authy/id494168017',
		),`

and, link text is "iOS: Google Authenticator". so i think 'iOS: "Google" Authenticator" link is shoud not link to 
the "authy"...

if provide 2 links, i think end user can select which.

Comment (ivankruchkoff) [reply google-authenticator.php]: you are correct @kassyi 

'link' => 'https://itunes.apple.com/app/google-authenticator/id388497605',

Will suffice

**Comment by ivankruchkoff** [comment]
I don't think you meant to close this

**Comment by kassyi** [comment]
sorry, I'm a beginner  at github. what's wrong?
I try to create new pull request, but this pull request open so can't create new pull request.

I make new pull request, so please pull this one.
https://github.com/ivankruchkoff/google-authenticator/pull/7

**Comment by ivankruchkoff** [comment]
Not a prob, I merged that PR. Thanks for your contribution @kassyi :)

**Comment by kassyi** [comment]
I'm happy to hear that! thx!

**Pull request closed without merging**